mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SurefireReportStorage.java

```diff
@@ -0,0 +1,92 @@
+package org.arquillian.smart.testing.mvn.ext;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Comparator;
+import org.apache.maven.execution.MavenSession;
+import org.apache.maven.model.Build;
+import org.apache.maven.model.Model;
+import org.arquillian.smart.testing.Logger;
+
+import static org.arquillian.smart.testing.spi.TestResult.TEMP_REPORT_DIR;
+
+class SurefireReportStorage {
+
+    static final String SUREFIRE_REPORTS_DIR_NAME = "surefire-reports";
+    private static final Logger logger = Logger.getLogger(SurefireReportStorage.class);
+
+    static void copySurefireReports(Model model) {
+        Build build = model.getBuild();
+        if (build != null && build.getDirectory() != null) {
+            File targetDir = new File(build.getDirectory());
+            if (targetDir.exists() && targetDir.isDirectory()) {
+                File[] surefireReports =
+                    targetDir.listFiles(file -> file.isDirectory() && SUREFIRE_REPORTS_DIR_NAME.equals(file.getName()));
+                if (surefireReports.length > 0) {
```

Comment:
Can it be more than 1?

Reply:
well, probably not. But it doesn't matter I guess - I just need to check if there is any, which means that the array is not empty